src/controllers/adminController.js

```diff
@@ -3,6 +3,17 @@ const Album = require('../models/Album');
 const adminHomePage = async (req, res) => {
   const allAlbums = await Album.find({}).sort({ price: 1 }); // Fetch all albums from the database and sort them by price in ascending order
   res.render('adminPage', { albums: allAlbums });
+  /*
+  #swagger.tags = ['Admin']
+
+  #swagger.summary = 'Admin Home Page'
+  #swagger.description = 'Fetch all albums and render the admin home page.'
+
+  #swagger.responses[200] = {
+    description: 'Admin home page fetched successfully.',
+    schema: { $ref: '#/definitions/Album' }
+  }
```

Comment:
Really the return value of this endpoint is not JSON of the albums, but rather an HTML page (eg: if you call this endpoint Postman with the proper credentials, you'll get the following):

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/5c82553d-92c4-4055-93c2-fbcca745e022)

<img width="1413" alt="Screenshot 2023-09-04 at 1 27 18 PM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/943d04af-958a-450b-9a70-bfe0a75d0a96">

<img width="1366" alt="Screenshot 2023-09-04 at 1 27 25 PM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/9516361d-a4da-4093-8f60-ffe1ae58c8be">

If we want to document that we'd need to do something like:

```
  #swagger.responses[200] = {
    content: { 'text/html': {} }
  }
```